src/main/java/com/autotune/common/k8sObjects/Metric.java

```diff
@@ -115,14 +58,11 @@ public String getValueType() {
         return valueType;
     }
 
-    @Override
-    public String toString() {
-        return "Metric{" +
-                "name='" + name + '\'' +
-                ", query='" + query + '\'' +
-                ", datasource='" + datasource + '\'' +
-                ", valueType='" + valueType + '\'' +
-                '}';
```

Comment:
Is it intentional to remove toString here?

Reply:
No . I just removed thinking not required at this point in time . But now I added it back